.github/workflows/publish-sphinx.yml

```diff
@@ -94,15 +94,15 @@ jobs:
         wget https://github.com/nexusformat/code/raw/master/applications/NXtranslate/docs/NXtranslate.pdf -O ./build/manual/source/_static/NXtranslate.pdf
 
     - name: Build (html) and Commit
-      uses: sphinx-notes/pages@master
+      uses: sphinx-notes/pages@main
       with:
         # path to conf.py directory
         documentation_path: build/manual/source
 
     - name: Publish if refs/tags
       # remove/comment next line to push right away
       if: startsWith(github.ref, 'refs/tags')
-      uses: ad-m/github-push-action@master
+      uses: ad-m/github-push-action@main
```

Comment:
not seeing a branch named `main` here: https://github.com/ad-m/github-push-action/branches